src/crux/api.clj

```diff
@@ -178,6 +178,10 @@
     from and including the valid time of the db while respecting
     transaction time. Includes the documents.")
 
+  (stats
```

Comment:
Is this a leftover perhaps? Feels like the stats makes sense on the system level, not on the db.